packages/flutter/lib/src/widgets/fade_in_image.dart

```diff
@@ -66,6 +74,12 @@ class FadeInImage extends StatelessWidget {
   /// Creates a widget that displays a [placeholder] while an [image] is loading,
   /// then fades-out the placeholder and fades-in the image.
   ///
+  /// To include a custom decode/cache size such as those provided by
```

Comment:
nit: This sentence is very complicated and hard to parse. What are you trying to say?

Reply:
Isn't this basically just saying that placeholder and image may also be ResizeImages?

Reply:
Added a simplified explanation